tests/contract/knative-channel-kafka/OWNERS

```diff
@@ -1,5 +1,5 @@
 reviewers:
-  - event-bus-reviewers
+  - knative-eventing-kafka-reviewers
```

Comment:
Would it make sense to take out `event-bus-reviewers` and `event-bus-approvers` completely? To be specific:
- resources/nats-streaming/OWNERS
- resources/knative-provisioner-natss/OWNERS
- resources/knative-eventing/OWNERS

Reply:
i wanted to consolidate all those changes in one PR